nixos/modules/services/backup/rsnapshot.nix

```diff
@@ -60,13 +62,28 @@ in
         cmd_ssh	${openssh}/bin/ssh
         cmd_logger	${inetutils}/bin/logger
         cmd_du	${coreutils}/bin/du
-        lockfile	/run/rsnapshot.pid
+        lockfile	${rundir}/rsnapshot.pid
 
         ${cfg.extraConfig}
       '');
     in {
       environment.systemPackages = [ myRsnapshot ];
 
+      systemd.services.rsnapshot.preStart = ''
```

Comment:
Lol. I copied it from another backup tool... but yeah, fixed it.

Btw: If you had annotated one line below the line you did annotate, the diff annotation would now vanish from the interface, as it would be outdated... :smile: